bot/cogs/snekbox.py

```diff
@@ -152,7 +157,7 @@ def get_status_emoji(results: dict) -> str:
         lines = output.count("\n")
 
         if lines > 0:
-            output = output.split("\n")[:10]  # Only first 10 cause the rest is truncated anyway
+            output = output.split("\n")[:11]  # Only first 11 cause the rest is truncated anyway
```

Comment:
Hardcoding and change this from `10` to `11` will lead to the same issue, just that you will be able to show 1 more line, should we think of a way to have a dynamic limitation ( show all if total characters are low ) for example? Otherwise, I can see a future where we will have another issue open for `bot only show 11 lines and truncate the 12th`.

Reply:
After a discussion in #dev-contrib, I agree that having 11 lines is fine since we also have a paste service. If this becomes a problem, we can raise another issue and handle it later.